itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -811,4 +819,120 @@ public void testIfCkptSetForObjectsByBootstrapReplLoad() throws Throwable {
     Partition uk = replica.getPartition(replicatedDbName, "t2", Collections.singletonList("uk"));
     verifyIfCkptSet(uk.getParameters(), tuple.dumpLocation);
   }
+
+  @Test
+  public void testIfCkptAndSourceOfReplPropsIgnoredByReplDump() throws Throwable {
+    WarehouseInstance.Tuple tuplePrimary = primary
+            .run("use " + primaryDbName)
```

Comment:
can add a test case, database created with source for replication property and then dump and load (to check create database handler)

Reply:
We cannot test Create Database handler as bootstrap doesn't use it. Also, cannot be part of incremental as well.